.asf.yaml

```diff
@@ -14,9 +14,5 @@ github:
     - mcu
     - microcontroller
   enabled_merge_buttons:
-    # enable squash button:
-    squash:  true
```

Comment:
Should we set these to false instead of relying on the default behaviour?

Reply:
I believe squashing can come in handy.
But if you want to keep it in the contributor's hand, that's okay with me.

Reply:
Done.

Reply:
> elieve squashing can come in handy.

We can ask contributor to do squash if need. In this way, contributor will learn how to organize the patch series in a better way and apply the knowledge in the furture PR.